packages/styles/src/utilities/_mixins.scss

```diff
@@ -39,34 +42,36 @@
     $state: if(map.has-key($utility, state), map.get($utility, state), ());
 
     $infix: if(
-      $property-class == '' and string.slice($infix, 1, 1) == '-',
+      $class-list == '' and string.slice($infix, 1, 1) == '-',
```

Comment:
Is this correct? Since `$class-list` is now a list I would imagine it is never equal to `''`